sky/clouds/service_catalog/gcp_catalog.py

```diff
@@ -30,12 +31,20 @@
 # TODO(zongheng): fix A100 info directly in catalog.
 # https://cloud.google.com/blog/products/compute/a2-vms-with-nvidia-a100-gpus-are-ga
 # count -> vm type
```

Comment:
BTW, maybe we can delete these comments now?

Reply:
Probably, not yet, as we still have the following dict in our code, instead of in the catalog. I think one action item here is to move the following to the catalog by having an entry combining A100 and its host machine like the catalog for the other clouds, i.e.:
```
InstanceType, AcceleratorName, AcceleratorCount, Price, SpotPrice, Region, AvailabilityZone
a2-highgpu-1g, A100, 8, 10, 2, us-west1, us-west1-a
```

Reply:
Oh that's the meaning of the comment. I misunderstood it.

I think it's fine to keep it as is, since the change would require several changes to our GCP provisioning logics.